### ISSUE TITLE
Setting Materials for OBJ

### ISSUE THREAD

**Issue by bboudaoud-nv** [NVlabs/abstract-fps#43]
The `amPose->materialTable.set()` calls in [App.cpp](https://github.com/NVlabs/abstract-fps/blob/master/target/source/App.cpp) uses an explicit OBJ name (`core/icosahedron_default`). This is a problem if we want to allow the [new target defined modelSpec](https://github.com/NVlabs/abstract-fps/commit/ce2b8bac414d75a93e4188ca75d13bd780908db3) to use OBJs without this internal name. For now our new cylinder model just uses this name internally. In the future it would be good to get this OBJ name rather than hard-coding it if possible.

**Comment by jspjutNV** [comment]
If we keep following our current method of having models that include a flipped border geometry outside, we are probably okay having a specific object mesh/material designed to be the object and another designed to be the silhouette. Since we started with a icosahedron target, `core/icosahedron_default` was the material, but we could go in and change that to a more logical name like `core/core` or something. 

I suggest we de-prioritize this fix until someone needs a wide variety of different target models since that would give us more information about how much we'd be able to modify the `.obj`s vs. just use `.obj`s made by someone else without modification.

**Comment by bboudaoud-nv** [comment]
I agree that this is not a high priority considering our (somewhat unique) approach to winding-order based geometry, I also agree that we should move to a more generic name (maybe `core/target`?). If you'd like we can keep this issue around as a reminder to transition to a more generic name.
